test/explorer/series_test.exs

```diff
@@ -2091,7 +2154,7 @@ defmodule Explorer.SeriesTest do
 
       result = Series.pow(s1, s2)
 
-      assert result.dtype == {:s, 64}
+      assert result.dtype == {:f, 64}
```

Comment:
Since we infer `Series.from_list([1, 2, 3])` to have type `{:s, 64}`, several result dtypes had to switch to float.